components/ContactUs/index.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="contact-us">
+    <div class="contact-us__main">
+      <div class="relative px-4 pt-[100px]">
+        <div class="flex flex-col gap-1">
+          <h3 class="contact-us__title">
+            Ingin Tahu Lebih Detail Tentang
+          </h3>
+          <h3 class="contact-us__title">
+            Desa Digital
+          </h3>
+        </div>
+        <div class="contact-us__cta">
+          <BaseButton class="contact-us__cta-btn">
+            <template #icon>
+              <img src="~/assets/icons/Whatsapp.svg" alt="Icon Whatsapp">
+            </template>
+            <template #label>
+              Hubungi Tim Kami Via Whatsapp
+            </template>
```

Comment:
minor:

maybe we can make `#label` to default slot? @adzharamrullah

Reply:
Maybe 😂

Reply:
here refactored slot label to default slot https://github.com/jabardigitalservice/dedi-landing-page/pull/9/commits/7cdc3b3458cea78247c5a072d87510c7a7efc563